.github/workflows/milestone-add-to-pr.yml

```diff
@@ -0,0 +1,33 @@
+# This action adds the latest milestone to a pull request
+# when it is merged
+
+name: 'Project: Add PR to Milestone'
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  if_merged:
+    if: github.event.pull_request.merged
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            const milestones = await github.rest.issues.listMilestones({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              state: "open"
+            })
+            for (const milestone of milestones.data) {
+              if (milestone.title == "next version") {
+                await github.rest.issues.update({
+                  owner: context.repo.owner,
+                  repo: context.repo.repo,
+                  issue_number: ${{ github.event.pull_request.number }},
```

Comment:
Could you use `context.issue.number` here?

Reply:
You could, i dont see much difference one way or the other